src/main/kotlin/org/jetbrains/research/testspark/tools/llm/generation/LLMProcessManager.kt

```diff
@@ -53,6 +58,10 @@ class LLMProcessManager(
     private val llmErrorManager: LLMErrorManager = LLMErrorManager()
     private val maxRequests = SettingsArguments.maxLLMRequest()
     private val testProcessor = TestProcessor(project)
+    private val sep = File.separatorChar
+
```

Comment:
Better to place such util-like variable closer to the place where it is needed, i.e. make it a local variable of a method.